### ISSUE TITLE
[enterprise-4.8] Issue in file networking/ingress-operator.adoc

### ISSUE THREAD

**Issue by hashnao** [openshift/openshift-docs#35774]
### Which section(s) is the issue in?
Configuring the PROXY protocol for an Ingress Controller

### What needs fixing?

The sample PROXY configuration of the hostNetwork configuration shows `type: NodePortService` but is it supposed to be `type: HostNetwork` instead?

```
  spec:
    endpointPublishingStrategy:
      hostNetwork:
        protocol: PROXY
      type: NodePortService
```

Hence, I think the Sample hostNetwork configuration to PROXY should look like this.

```
  spec:
    endpointPublishingStrategy:
      hostNetwork:
        protocol: PROXY
      type: HostNetwork
```

**Comment by jboxman-rh** [comment]
cc @skrthomas

**Comment by skrthomas** [comment]
@jboxman-rh looks like @stevsmit fixed this issue with #36526 so it can be closed now.

**Comment by jboxman-rh** [comment]
@skrthomas thanks!
